.lgtm/cpp-queries/PotentiallyDangerousFunction.ql

```diff
@@ -0,0 +1,30 @@
+/**
+ * @name Use of potentially dangerous function
+ * @description Certain standard library functions are dangerous to call.
+ * @kind problem
+ * @problem.severity error
+ * @precision high
+ * @id cpp/potentially-dangerous-function
+ * @tags reliability
+ *       security
+ *
+ * Borrowed from
+ * https://github.com/Semmle/ql/blob/master/cpp/ql/src/Security/CWE/CWE-676/PotentiallyDangerousFunction.ql
```

Comment:
The query pack has just been updated and everything seems to be more or less fine except that "borrowed", "from" and the url are considered tags: https://lgtm.com/rules/1507954176130/. @samlanning is this expected and I should move that comment somewhere else?

It's unrelated, but I've just ran into https://lgtm.com/blog/introducing_dataflow_path_exploration. I'm wondering if it is the same as "taint source modeling" you were talking about in https://github.com/systemd/systemd/pull/11754#issuecomment-465707129.

Reply:
@evverx regarding the tags, yes this is expected. For comments like that, i would suggest just using 'normal' block comments outside of the `QLDoc`, underneath it. i.e. just using `/* */`

Regarding the taint source modelling, the resulting query would probably be a taint-tracking or data-flow query, so yes. The query could be expressed as a `path-problem`, and would produce paths that can be viewable in LGTM's UI.